tawk_to.module

```diff
@@ -206,3 +206,15 @@ function tawk_to_remove_widget() {
 
   backdrop_json_output(array('success' => true));
 }
+
+
+/**
+ * Implements hook_config_info().
+ */
+function tawk_to_config_info() {
+    $info[TawkHelper::TAWK_TO_CONFIG_NAME] = array(
```

Comment:
@yorkshire-pudding is `$info` here always defined? Might be better to add a fail-safe here so that it won't cause any 
 `undefined` error. Also, there are extra spaces at the beginning of the line.

Reply:
NVM on the `$info` part. Though, I did take a look at the documentation and some examples on the hook. I found that instead of using `$info`, they're using the `$prefixes` variable to set the configuration info. I think it would be better if we can also do the same so we can be uniform with the others.

Ref:
- https://docs.backdropcms.org/api/backdrop/core%21modules%21config%21config.api.php/function/hook_config_info/1
- https://docs.backdropcms.org/api/backdrop/core%21modules%21admin_bar%21admin_bar.module/function/admin_bar_config_info/1
- https://docs.backdropcms.org/api/backdrop/modules%21examples%21ajax_example%21ajax_example.module/function/ajax_example_config_info/1
- https://docs.backdropcms.org/api/backdrop/core%21modules%21block%21block.module/function/block_config_info/1
- https://docs.backdropcms.org/api/backdrop/modules%21examples%21action_example%21action_example.module/function/action_example_config_info/1

Reply:
Thanks for reviewing this @alvinjohnsonso - I've looked at the docs and you're right that in core modules they are using prefixes.  I was basing it on another contrib module that I'd recently found a config bug in, but happy to make it consistent with core.  I've corrected the spacing at the beginning of each line.